api/generated/common/types.go

```diff
@@ -79,6 +79,63 @@ type AccountParticipation struct {
 	VoteParticipationKey []byte `json:"vote-participation-key"`
 }
 
+// Application defines model for Application.
+type Application struct {
+
+	// \[appidx\] application index.
+	AppIndex uint64 `json:"app-index"`
```

Comment:
should we keep it consistent with assets? `app-index` vs `index`. they're both should be probably just `id` given the context.

Reply:
Yes, definitely.